packages/m2m-gateway/test/api/eservices/createEServiceRiskAnalysis.test.ts

```diff
@@ -59,7 +59,7 @@ describe("POST /eservice/:eserviceId/riskAnalyses router test", () => {
         mockRiskAnalysisSeed
       );
 
-      expect(res.status).toBe(201);
+      expect(res.status).toBe(200);
```

Comment:
In my previous comment, I meant to update the test description to 201, the code check was right :)
```suggestion
      expect(res.status).toBe(201);
```